microsoft-edge/web-platform/site-impacting-changes.md

```diff
@@ -19,10 +19,11 @@ The web is constantly evolving to improve the user experience, security, and pri
 | Display subtle prompt for notification permissions requests | Edge v84 |  | Quiet notification requests display a subtle request icon in the address bar for site notification permissions requested using the `Notifications` or `Push` API, replacing the full or standard permission flyout prompt UI.  This feature is currently enabled for all users.  To opt out of quiet notification requests, go to `edge://settings/content/notifications`.  In the future, the Microsoft Edge team may explore re-enabling the full flyout notification prompt in some scenarios.  |  
-| Disable TLS/1.0 and TLS/1.1 by default | Edge v84 |  | To help discover impacted sites, you may set the `edge://flags/#display-legacy-tls-warnings` flag to cause Microsoft Edge to display a non-blocking "Not Secure" notice when loading pages that require legacy TLS protocols.  The [SSLMinVersion][DeployedEdgePoliciesSSLMinVersion] Group Policy permits re-enabling of TLS/1.0 and TLS/1.1; the policy remains available until Edge 88.  |  
+| Disable TLS/1.0 and TLS/1.1 by default | Edge v84 |  | The [SSLMinVersion][DeployedEdgePoliciesSSLMinVersion] Group Policy permits re-enabling of TLS/1.0 and TLS/1.1; the policy will remain available until Edge 90.  |  
 | Block mixed content downloads | [Chrome+1](#release-comments) \(Edge v86\)  |  | This change is happening in the Chromium project, on which Microsoft Edge is based.  For more information, including the planned timeline by Google for this change, please review the [Google security blog entry][GoogleBlogSecurity20200206].  The Microsoft rollout schedule on file types to warn or block is planned for one release after Chrome.  |  
 | Deprecate AppCache | [Chrome+1](#release-comments) \(Edge v86\)  |  | This change is happening in the Chromium project, on which Microsoft Edge is based.  For more information, please review the [WebDev documentation][WebDevAppCacheRemoval].  The Microsoft rollout schedule for deprecation is planned for one release after Chrome.  Requesting an [AppCache OriginTrial Token][AppCacheOriginTrial] allows sites to continue to use the deprecated API until Edge v90.  |  
 | Removal of Adobe Flash | Edge v88  |  | This change is happening in the Chromium project, on which Microsoft Edge is based.  For more information, please review the [Adobe Flash Chromium Roadmap][ChromiumFlashRoadmapSupportRemoved].  | 
+| Disable and remove FTP | Edge v88  | Edge v87 | In v87, FTP support will be disabled by default. In v88, FTP support will be removed. This change is happening in the Chromium project, on which Microsoft Edge is based. For more information, please review the [Chrome Platform Status Entry][[ChromePlatformStatus6246151319715840].  | 
```

Comment:
```suggestion
| Turn off and remove FTP | Edge v88  | Edge v87 | In v87, FTP support will be turned off by default. In v88, FTP support will be removed. This change is happening in the Chromium project, on which Microsoft Edge is based. For more information, please review the [Chrome Platform Status Entry][[ChromePlatformStatus6246151319715840].  | 
```

Reply:
If we prefer "Turn off" over "Disable" for whatever reason that's fine with me, but we should probably change the other uses in the document for consistency?

Reply:
Sure - turn off or remove seem to be good options for the other sentences where it was used. Can you update the other instances in the doc, or would you like me to take care of that? Thanks.

Reply:
Updated.